### ISSUE TITLE
Size of training dataset?

### ISSUE THREAD

**Issue by xiankun1995** [Microsoft/O-CNN#29]
Hello. I have some questions again in segmentation task. 
First is how many 3d models I should use In the training stage. I want to find fit numbers of models.
And second is whether the input is only 3d models with segmentation labels.
Thank you for your help.

**Comment by wang-ps** [comment]
1. According to our experiment, several hundreds of models are enough. Of course, the more the better.
2. I can not fully understand your second question.

**Comment by xiankun1995** [comment]
Sorry let me rephrase my 2nd question. To train o-cnn for semantic segmentation task, I only need the 3D segmented models as the input, is that right?

**Comment by wang-ps** [comment]
Yes
